dialogs.py

```diff
@@ -3,8 +3,13 @@
 from functools import partial
 
 
-def popup(parent, title, details, icon, *, buttons):
+def popup(parent, title, details, icon, *, buttons, theme):
     dialog = tk.Toplevel()
+    window.tk.call("source", "sun-valley.tcl")
+    
+    window.tk.call("set_theme", "%s" % theme)
```

Comment:
```suggestion
```
This breaks when showing a popup more than once. The theme can only be loaded once.
Please also remove the `theme` option.